src/components/app/details/main.tsx

```diff
@@ -114,9 +115,6 @@ export function AppHeader() {
         } catch (err) {
             showError(err)
         }
-        return () => {
```

Comment:
is it intentional?

Reply:
Yes i was adding some hack to render. but forget to remove earlier